_config.yml

```diff
@@ -75,6 +75,8 @@ collections:
     output: true
   lovelace:
     output: true
+  configuration:
```

Comment:
Why is this pulled out into a separate collection?

Reply:
When does it make sense to have a separate collection and when do we put it into the docs folder?
locelace and addons are a good example for this.
Also the hassio folder even though it is not a collection.

Is there a reason for that?

Reply:
You've put it into a collection? Not me. I wonder why? Could you elaborate on that? Why is that needed?